docs/functions.rst

```diff
@@ -150,6 +150,82 @@ Example:
         - |
           echo "Example"
 
+KubeTaskParallel
+----------------
+
+KubeTaskParallel spins up a new pod with two containers connected
+via shared `emptyDir`_ volume.
+It's similar to KubeTask, but allows using multiple images to move backup data.
+"background" container is one responsible for generating data, while "output" container
+should export it to destination.
+The main difference between them is that phase outputs can only generated from the
+"output" container outputs.
+The function also supports an optional init container to set up the volume contents.
+
+.. csv-table::
+   :header: "Argument", "Required", "Type", "Description"
+   :align: left
+   :widths: 5,5,5,15
+
+   `namespace`, No, `string`, namespace in which to execute (the pod will be created in controller's namespace if not specified)
+   `backgroundImage`, Yes, `string`, image to be used in "background" container
+   `backgroundCommand`, Yes, `[]string`,  command list to execute in "background" container
```

Comment:
should we call this container snapshot or backup container? or maybe data container? just a suggestion, if you think `background` is ok, I am good.